altair/utils/save.py

```diff
@@ -128,7 +148,7 @@ def save(
             **kwargs,
         )
         write_file_or_filename(fp, mimebundle["text/html"], mode="w")
-    elif format in ["png", "svg", "pdf"]:
+    elif format in ["png", "svg", "pdf", "vega"]:
```

Comment:
I can't comment on the correct line but `vega` does not work here. It will try to access the mimebundle with key `image/svg+xml` on line 169. 

Could also add support for "vega-lite" here?

Reply:
`vega-lite` cannot be supported as `format`. That would be `json`. `format` functions a bit like export to, or save as in other programs. If it is defined as `vega` it should compile the vega-lite specification into vega json.
Apparently that never has functioned.

Reply:
Of course, you're right, that would be `json`!

Reply:
Hm, I just noticed the following variations in the code:
https://github.com/altair-viz/altair/blob/1ed3f6f2fd3647bcf1f2e5e81dd328a0f9cf623d/altair/utils/mimebundle.py#L72-L77

It seems there is different type of json, namely, `application/json` and `application/vnd.vegalite.v5+json` (for altair 5). So I'll include `vega-lite` also as `format`, it probably would be better if it was renamed to something as `vl-json`.